solid-logo-usage-guidelines.md

```diff
@@ -1,17 +1,176 @@
 [![](https://img.shields.io/badge/project-Solid-7C4DFF.svg?style=flat-square)](https://github.com/solid/solid)
 
+# Introduction
+
+These guidelines are to help you when representing the Solid brand in apps, communications, content, events, projects or assets. They incorporate guidance for visual elements such as logos, colours, typography, iconography and graphic devices. There is also guidance for writing such as tone and format. These guidelines are intended for digital application. For more specific print guidelines please contact [Mitzi László](https://github.com/Mitzi-Laszlo) for further guidance.
+
+Why do we need brand guidelines?
+-
+As a defined entity, Solid has an identity. The more consistent that identity is the easier it is to be identified and identify with. While the Solid brand may appear in a range of different places for very different purposes, by following these guidelines you will be helping to retain that consistency and build a stronger identity.
+
+For app developers
+-
+Please use these guidelines for any instances where you are looking to reference Solid. In particular, please pay notice to the guidelines on logo and colour usage as well as how to refer to Solid and its key elements in text.
+
+For event organisers or communications/content creators
+-
+If you are looking to create Solid communications, collateral or content please follow all aspects of these guidelines. For guidance on creating user interfaces, videos, animation or sound please contact [Mitzi László](https://github.com/Mitzi-Laszlo) for further guidance.
+___
+
+# Values
```

Comment:
Should we consolidate this section with the [values document](https://github.com/solid/information/blob/master/solid-values.md)? @Mitzi-Laszlo

Reply:
let's make a link so that it doesn't diverge